tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -1177,3 +1177,75 @@ describe("Test Application deployment based on 'clusterGroupSelector'", { tags:
   })
 });
 
+if (!/\/2\.7/.test(Cypress.env('rancher_version')) && !/\/2\.8/.test(Cypress.env('rancher_version'))){
+
+  describe('Test namespace deletion when bundle is deleted', { tags: '@p1'}, () => {
+    
+    qase(131,
+      it("Fleet-131: Test NAMESPACE will be DELETED after GitRepo is deleted.", { tags: '@fleet-131' }, () => {
+        const repoName = 'test-ns-deleted-when-bundle-deleted'
+        const namespaceName = 'my-custom-namespace'
+
+        cy.fleetNamespaceToggle('fleet-local');
+        cy.clickButton('Add Repository');
+        cy.clickButton('Edit as YAML');
+        cy.addYamlFile('assets/131-ns-deleted-when-bundle-deleted.yaml');
+        cy.clickButton('Create');
+        cy.checkGitRepoStatus(repoName, '1 / 1', '1 / 1');
+
+        // Check namespace is created 
+        cy.accesMenuSelection('local', 'Projects/Namespaces');
+        cy.filterInSearchBox(namespaceName);
+        cy.verifyTableRow(0, 'Active', namespaceName);
+
+        // Delete GitRepo
+        cy.deleteAllFleetRepos();
+
+        // Check namespace is deleted
+        cy.accesMenuSelection('local', 'Projects/Namespaces');
+        cy.filterInSearchBox(namespaceName);
+        cy.contains(namespaceName).should('not.exist');
+      })
+    )
+
+    qase(131,
```

Comment:
Same test case name?

Reply:
Yes, this is what I was not sure how to address.
I have 2 scenarios under same test case.
How would you address it?

Reply:
Addressed splitting test cases into different qase ones